components/base/base.go

```diff
@@ -49,13 +49,15 @@ type Base interface {
 
 	// Spin spins the robot by a given angle in degrees at a given speed.
 	// If a speed of 0 the base will stop.
+	// A positive angle is measured in degrees to the left.
+	// A positive speed turns to the left.
 	// This method blocks until completed or cancelled
 	Spin(ctx context.Context, angleDeg, degsPerSec float64, extra map[string]interface{}) error
 
 	SetPower(ctx context.Context, linear, angular r3.Vector, extra map[string]interface{}) error
 
-	// linear is in mmPerSec
-	// angular is in degsPerSec
+	// linear is in mmPerSec (positive Y moves forwards)
+	// angular is in degsPerSec (positive Z turns to the left)
```

Comment:
```suggestion
	// angular is in degsPerSec (positive Z turns to the left for built-in RDK drivers)
```
@randhid here too yeah?

Reply:
yep!